src/abi/tests.rs

```diff
@@ -822,7 +921,7 @@ contract Testing {
 
     assert_eq!(idl.instructions[2].name, "getNum_int32_int32");
     assert!(idl.instructions[2].docs.is_none());
-    assert!(idl.instructions[2].accounts.is_empty());
+    assert_eq!(idl.instructions[2].accounts.len(), 1);
```

Comment:
How about checking if we have the right account?

Reply:
Fixed